crates/node/service/src/actors/derivation.rs

```diff
@@ -118,24 +142,14 @@ impl PipelineBuilder for DerivationBuilder {
 
 /// The outbound channels for the derivation actor.
```

Comment:
It looks like this is now over `DerivationInboundChannels` thought github diff sometimes renders weird. Should this be updated to `inbound` channels?

```suggestion
/// The inbound channels for the derivation actor.
```